pages/3_⚖️_Environmental_Justice.py

```diff
@@ -72,21 +70,75 @@ def add_spatial_data(url, name, projection=4326):
+  "UNEMPPCT": "Percent of a block group's population that did not have a job at all during the reporting period, made at least one specific active effort to find a job during the prior 4 weeks, and were available for work (unless temporarily ill).",
+  #"VULEOPCT": "Demographic index of block group used by EPA, an average of low income and people of color populations",
+  #"DISPEO": "Intermediate variable used for calculation of demographic index",
+  #"DSLPM": "Diesel particulate matter level in air, µg/m3",
+  #"CANCER": "Lifetime cancer risk from inhalation of air toxics",
+  #"RESP": "Ratio of exposure concentration to health-based reference concentration",
+  #"PTRAF": "Count of vehicles (AADT, avg. annual daily traffic) at major roads within 500 meters, divided by distance in meters (not km)",
+  #"PWDIS": "RSEI modeled toxic concentrations at stream segments within 500 meters, divided by distance in kilometers (km)",
+  #"PNPL": "Count of proposed or listed NPL - also known as superfund - sites within 5 km (or nearest one beyond 5 km), each divided by distance in kilometers",
+  #"PRMP": "Count of RMP (potential chemical accident management plan) facilities within 5 km (or nearest one beyond 5 km), each divided by distance in kilometers",
+  #"PTSDF": "Count of hazardous waste facilities (TSDFs and LQGs) within 5 km (or nearest beyond 5 km), each divided by distance in kilometers",
+  #"OZONE": "Annual average of top ten maximum daily 8-hour ozone air concentrations in parts per billion",
+  #"PM25": "PM2.5 levels in air, µg/m3 annual avg.",
+  #"UST": "Count of leaking underground storage tanks (multiplied by a factor of 7.7) and the number of underground storage tanks within a 1,500-foot buffered block group"
+}
+options = ejdefs.values()
+
 # Load and join census data
 with st.spinner(text="Loading data..."):
   census_data = add_spatial_data(url="https://www2.census.gov/geo/tiger/TIGER2017/BG/tl_2017_34_bg.zip", name="census") #, projection=4269
   ej_data = pd.read_csv("https://github.com/edgi-govdata-archiving/ECHO-SDWA/raw/main/EJSCREEN_2021_StateRankings_NJ.csv") # NJ specific
-  ej_data["ID"] = ej_data["ID"].astype(str)
-  census_data.set_index("GEOID", inplace=True)
-  ej_data.set_index("ID", inplace=True)
-  census_data = census_data.join(ej_data)
+  ej_data["ID"] = ej_data["ID"].astype(str) # set the Census id as a string
+  census_data.set_index("GEOID", inplace=True) # set it to the index in the Census data
+  ej_data.set_index("ID", inplace=True) # set the Census id to the index in the EJScreen data
+  census_data = census_data.join(ej_data) # join based on this shared id
+  census_data.rename(columns=ejdefs, inplace=True) # rename columns to the EJ variable definitions
+  census_data = census_data[[i for i in ejdefs.values()] + ["geometry"]] # Filter out unnecessary columns
+  census_data[[i for i in ejdefs.values()]] = round(census_data[[i for i in ejdefs.values()]] * 100, 2) # Convert decimal values to percentages
```

Comment:
Also need to append "%" before displaying in the map

Reply:
Done